src/Entities/ApplicantEntity.php

```diff
@@ -96,6 +98,12 @@ private function sanitiseData()
         $this->eighteenPlus = $this->eighteenPlus ? 1 : 0;
         $this->finance = $this->finance ? 1 : 0;
         $this->notes = StringValidator::sanitiseString($this->notes);
+        $this->name = StringValidator::validateExistsAndLength($this->name, 255);
```

Comment:
can you put 255 and 10000 into class constants so that they are not duplicated values